.github/workflows/publish-bundle.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish bundle to CharmHub
+on:
+  workflow_call:
+    inputs:
+      release:
+        description: Bundle release to run tests on and publish to Charmhub
+        type: string
+        required: true
+    secrets:
+      CHARMCRAFT_CREDENTIALS:
+        required: true
+  workflow_dispatch:
+    inputs:
+      release:
+        description: Bundle release to run tests on and publish to Charmhub (i.e. 1.10/stable
+        type: string
+        required: true
+    secrets:
+      CHARMCRAFT_CREDENTIALS:
+        required: true
+
+jobs:
+  publish-bundle-for-releases-affected:
```

Comment:
The job name could be updated too (`publish-bundle`). "-for-releases-affected" looks like a leftover.